components/floatingActionButton.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+*Contributors: Jonny Yang
+ */
+
+import { Ionicons } from "@expo/vector-icons";
+import { StyleSheet, TouchableOpacity, ViewStyle } from "react-native";
+
+interface FloatingActionButtonProps {
```

Comment:
Use `type` instead of `interface` for consistency

```suggestion
type FloatingActionButtonProps {
```